enterprise/backend/test/metabase_enterprise/sso/integrations/sso_utils_test.clj

```diff
@@ -9,9 +9,11 @@
       "/test"
       "localhost"
       "localhost:3000"
-      "http://localhost:3000"))
+      "http://localhost:3000"
+      "http://localhost:3000/dashboard/1-test-dashboard?currency=British%20Pound"))
 
   (testing "check-sso-redirect- throws an error for invalid redirect URIs"
     (are [uri] (thrown-with-msg? clojure.lang.ExceptionInfo #"Invalid redirect URL" (sso-utils/check-sso-redirect uri))
       "http://example.com"
```

Comment:
are you able to run this test locally with a fresh DB? I tried and it's currently fail. I am not sure why it does not fail on CI.

Reply:
I think we need to bind `public-settings/site-url`

Reply:
Hadn't tried with a fresh DB. I think in CI probably a previous test populates the value of `site-url` somewhere. Good idea to set it explicitly in the test though